src/defs/server.ts

```diff
@@ -1256,15 +1237,6 @@ tmpDef.argDescs = {};
 tmpDef.scriptSide = ScriptSide.Server;
 ServerDefinitions.push(tmpDef);
 
-tmpDef = new MTAFunction;
-tmpDef.label = "setPlayerHudComponentVisible";
```

Comment:
Same here. Since it basically is the same, but has different parameters (which the dupe check doesnt catch) this should be kept.